File: ControlRemote/Web/Dto/ActionSortByDateTimeModel.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace Web.Dto
{
    public class ActionSortByDateTimeModel
    {
        public DateTime DateTimeAction { get; set; }
        public List<ActionPointAtTimeModel> Commands { get; set; }
    }
}

File: ControlRemote/Web/ClientApp/src/app/Dto/ActionSortByDateTimeModel.ts
import { ActionPointAtTimeModel } from "./ActionPointAtTimeModel";

export class ActionSortByDateTimeModel {
    dateTimeAction: Date;
    commands: ActionPointAtTimeModel[];

    public constructor(_dateTimeAction: Date, _commands: ActionPointAtTimeModel[]) {
        this.dateTimeAction = _dateTimeAction;
        this.commands = _commands;
    }
}